lib/ansible/modules/network/nxos/nxos_ping.py

```diff
@@ -75,7 +75,7 @@
     host: 68.170.147.165
   with_items:
     - 8.8.8.8
```

Comment:
what about Google DNS then?

Reply:
a) This PR in no way expects to be comprehensive. #38959 lists a lot more instances, and they were only looking for `1.1.1.1`. I'm just doing a best effort search-and-replace with sed in the networking code.
b) That test is specifically looking for IPs which _are_ routeable (and thanks for pointing that out as I need to revert the change to that test as a result), so Google DNS is in fact what we want here